lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -8375,15 +8413,15 @@ namespace Js
                 // Else, set it to next element after current nextEvictionVictim index
                 cache->nextEvictionVictim = (cache->nextEvictionVictim + 1) & (EQUIVALENT_TYPE_CACHE_SIZE - 1);
             }
-
+   
             if (PHASE_TRACE1(Js::EquivObjTypeSpecPhase))
             {
-                Output::Print(_u("EquivObjTypeSpec: Saving type in used slot of equiv types cache at index = %d. NextEvictionVictim = %d. \n"), index, cache->nextEvictionVictim);
+                Output::Print(_u("Evicted a type from equiv type cache because of a non shared dictionary type"));
```

Comment:
>Output::Print(_u("Evicted a type from equiv type cache because of a non shared dictionary type")); [](start = 16, length = 98)

why did you change this?